src/main/webapp/app/exercises/quiz/participate/quiz-participation.component.ts

```diff
@@ -132,6 +133,7 @@ export class QuizParticipationComponent implements OnInit, OnDestroy {
         private quizExerciseService: QuizExerciseService,
         private participationService: ParticipationService,
         private route: ActivatedRoute,
+        private router: Router,
```

Comment:
Was it an active decision not use the more explicit _inject_ here? 

`private readonly router = inject(Router);`